libs/lib-client/header/src/lib/header.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
```

Comment:
can we add documentation of usage? Just a short line and an example of usage, mostly because we're using transclusion (`ng-content`) and coming back later to this it would be useful to have a quick glance at how this component is supposed to work:

For example:

```ts
/**
 * <Description>
 * 
 * Customizable areas:
 *  - resource logo (selector: `resource-logo`): ...
 * 
 * Example of usage:
 *  <flogo-designer-header>
 *   <div header-resource-logo><svg>logo to display</svg></div>
 *   ...
 *  </flogo-designer-header>
**/
```

We can also add more examples/details in a readme in the root of this library, like the one in modal: https://github.com/project-flogo/flogo-web/blob/master/libs/lib-client/modal/README.md

Reply:
Did not add the readme but I added the documentation to the component's definition